src/components/PostsList.tsx

```diff
@@ -40,7 +40,7 @@ export const PostsList: React.FC<Props> = ({
                   'is-light': post.id !== selectedPostId,
```

Comment:
This line correctly uses the 'classNames' library to conditionally apply the 'is-light' class based on whether the post is selected. However, ensure that the 'selectedPostId' is correctly managed and updated elsewhere in your application to reflect the current state.